UPCarouselFlowLayout/UPCarouselFlowLayout.swift

```diff
@@ -60,7 +59,7 @@ open class UPCarouselFlowLayout: UICollectionViewFlowLayout {
         
         let yInset = (collectionSize.height - self.itemSize.height) / 2
         let xInset = (collectionSize.width - self.itemSize.width) / 2
-        self.sectionInset = UIEdgeInsetsMake(yInset, xInset, yInset, xInset)
+        self.sectionInset = UIEdgeInsets.init(top: yInset, left: xInset, bottom: yInset, right: xInset)
```

Comment:
I think the `.init` part shouldn't be here, as it looks like the isn't used in other places.